sdks/python/tox.ini

```diff
@@ -80,12 +80,12 @@ commands =
 [testenv:py27-lint]
 deps =
   pycodestyle==2.3.1
-  pylint==1.7.2
+  pylint==1.9.3
   future==0.16.0
   isort==4.2.15
   flake8==3.5.0
 commands =
-  python --version
+  pylint --version
```

Comment:
Instead of removing "python --version", can you add "pylint --version" as a new line instead?

Reply:
The output would be redundant because __pylint --version__ is an effective superset of __python --version__

$ __python --version__
Python 3.7.0
$ __pylint --version__
pylint 2.1.1
astroid 2.0.4
Python 3.7.0 (default, Jun 29 2018, 20:13:13)
[Clang 9.1.0 (clang-902.0.39.2)]